client/components/Coverages/CoverageIcon.tsx

```diff
@@ -85,7 +85,7 @@ export class CoverageIcon extends React.PureComponent<IProps> {
                         planningUtils.getCoverageIcon(
                             planningUtils.getCoverageContentType(this.props.coverage, this.props.contentTypes) ||
                             this.props.coverage.planning?.g2_content_type,
-                            this.props.coverage
+                            this.props.coverage,
```

Comment:
This change can be removed, as it does nothing to the code and there are no other changes in this file